invenio_app_ils/records/resolver/jsonresolver/document_circulation.py

```diff
@@ -24,8 +25,14 @@ def jsonresolver_loader(url_map):
 
     def circulation_resolver(document_pid):
         """Return circulation info for the given Document."""
+        rest_cfg = current_app.config["CIRCULATION_REST_ENDPOINTS"]
+
+        loan_search = current_circulation.loan_search
+
+        record_cfg = current_app.config["RECORDS_REST_ENDPOINTS"]
+        ItemSearch = obj_or_import_string(
+            record_cfg[ITEM_PID_TYPE]["search_class"])
```

Comment:
Whenever you see you can fix also any other hardcoded reference for the Search classes, please do!
Feel free to add them to the [ext here](https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/ext.py#L39) as done for the indexers.